BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,284 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class ProductPage extends BaseClass {
+
+	/**
+	 * Constructor
+	 */
+	public ProductPage() {
+		PageFactory.initElements(driver, this);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(css = "#tab-menu-8651")
+	private WebElement prodHeaderMenu;
+
+	@FindBy(css = "[itemprop='name'] > #node-title-url-386")
+	private WebElement prodTileTitle;
+
+	@FindBy(css = "[alt='BENADRYL® Allergy ULTRATABS® Tablets']")
+	private WebElement prodImg;
+
+	@FindBy(css = "#content-main")
+	private WebElement prodTitle;
+
+	@FindBy(css = ".bv_ratings_summary")
+	private WebElement prodRating;
+
+	@FindBy(css = ".bv_ratings_summary > [itemprop='ratingValue']")
+	private WebElement prodRatingNum;
+
+	@FindBy(css = ".inside > .pane-node-body")
+	private WebElement prodOverview;
+
+	@FindBy(css = ".inside > .pane-node-body > div > div > div > p:nth-child(1)")
+	private WebElement prodOverviewTxt;
+
+	@FindBy(css = ".bv_war_button")
+	private WebElement writeAReviewBtn;
+
+	@FindBy(css = ".zZXQY")
+	private WebElement writeAReviewModal;
+
+	@FindBy(xpath = "(//button[@aria-label='Find where to buy this product'])[1]")
+	private WebElement buyNowBtn;
+
+	@FindBy(css = ".ps-wtb")
+	private WebElement buyNowModal;
+
+	@FindBy(xpath = "//h2[normalize-space()='Benadryl Allergy Ultratabs Tablets, 48 Count']")
+	private WebElement buyNowModalHeading;
+
+	@FindBy(css = ".jump-to-title")
+	private WebElement jumpToTitle;
+
+	@FindBy(css = "[href=\"#product-overview\"] > h2")
+	private WebElement prodOverviewHeading;
+
+	@FindBy(css = ".pane-node-field-product-directions")
+	private WebElement directionSection;
+
+	@FindBy(css = ".pane-node-field-product-directions h2")
+	private WebElement directionHeading;
+
+	@FindBy(id = "mini-panel-product_ingredients_modal")
+	private WebElement ingredientsSection;
+
+	@FindBy(css = "#mini-panel-product_ingredients_modal h2")
+	private WebElement ingredientsHeading;
+
+	@FindBy(css = "li #used-for")
+	private WebElement usedForSection;
+
+	@FindBy(id = "tab_used-for")
+	private WebElement usedForHeading;
+
+	@FindBy(css = "li #warnings")
+	private WebElement warningsSection;
+
+	@FindBy(id = "tab_warnings")
+	private WebElement warningsHeading;
+
+	@FindBy(id = "mini-panel-product_faq")
+	private WebElement faqsSection;
+
+	@FindBy(css = "#mini-panel-product_faq span")
+	private WebElement faqsHeading;
+
+	@FindBy(css = "#bv-product-reviews")
+	private WebElement reviewsSection;
+	
+	/**
+	 * Returns the WebElement of a 'Jump To' heading based on its text.
```

Comment:
@zahinSJI no need to add description for locators

Reply:
Descriptions removed for locators